zephyr/CMakeLists.txt

```diff
@@ -712,6 +712,31 @@ zephyr_library_sources_ifdef(CONFIG_COMP_COPIER
 	${SOF_AUDIO_PATH}/copier.c
 )
 
+zephyr_library_sources_ifdef(CONFIG_MAXIM_DSM
+	${SOF_AUDIO_PATH}/smart_amp/smart_amp.c
+	${SOF_AUDIO_PATH}/smart_amp/smart_amp_generic.c
+	${SOF_AUDIO_PATH}/smart_amp/smart_amp_maxim_dsm.c
+)
+
+zephyr_library_sources_ifdef(CONFIG_COMP_ARIA
+	${SOF_AUDIO_PATH}/aria/aria.c
+)
+
+zephyr_library_sources_ifdef(CONFIG_COMP_CROSSOVER
+	${SOF_AUDIO_PATH}/crossover/crossover.c
+	${SOF_AUDIO_PATH}/crossover/crossover_generic.c
+)
+
+zephyr_library_sources_ifdef(CONFIG_COMP_DRC
+	${SOF_AUDIO_PATH}/drc/drc.c
+	${SOF_AUDIO_PATH}/drc/drc_generic.c
+)
+
+zephyr_library_sources_ifdef(CONFIG_COMP_MULTIBAND_DRC
+	${SOF_AUDIO_PATH}/multiband_drc/multiband_drc.c
+	${SOF_AUDIO_PATH}/multiband_drc/multiband_drc_generic.c
```

Comment:
These really do not look like sparse fixes.

Reply:
@marc-hb I have to update drc and other components for cache / sparse and that only works with Zephyr, so I need to add them to Zephyr builds

Reply:
I'm sure you know what you're doing but this seems really buried in a large PR that seems to be mostly about something else. It's not mentioned in the description and I could not find a commit message mentioning it either?